### ISSUE TITLE
Multiple Playback Speed settings and hotkeys are Gone?

### ISSUE THREAD

**Issue by EroAxe** [GDQuest/blender-power-sequencer#472]
**I'm submitting a...**

- [ ] Bug report.
- [x] Feature request.

## Bug report ##

**What is the current behavior?**

**What is the expected behavior?**

**Tell us the steps to reproduce the bug, and if possible share a minimal demo of the problem.** 

## Feature request ##

I'm making this to request/find some information about why the 1.3x and 1.6x playback speed options, as well as the hotkeys for changing the speed disappeared from the addon awhile ago.  I used them quite heavily since 2x speed is generally waaay faster than I need or want, most of the time.

**Describe the problem you're trying to solve.**

The ability to playback footage at a higher or lower speed to streamline the editing process without needing to repeatedly move backwards on the timeline.

**Tell us which solutions you've explored, the solution you would pick, and why you think it would be the best for everyone.**

I've attempted to bind the existing options from the addon in Blender to get back the hotkeys, but unfortunately that only lets me bind to one specifically.  Not just an "increase" or "decrease".

As for the other options I've checked other addons to see if it's a supported feature and it's not.  There are some great options for those addons, but nothing for playback speed.

When it comes to a solution I would pick, the simplest would probably just be, what we had before.  Where there were the hotkeys, you could bind them, and you had more fine control over playback speed.  If possible I'd hope for more options that are a bit more granular, maybe going up by 100s, ex. 1.1, 1.2 but that's just hoping.

## Other ##

**Other information** 

The hotkeys and the extra options are even still listed on the [documentation](https://gdquest.gitbook.io/blender-power-sequencer/features-reference) for context, so I'm a little confused as to where they went.

**Comment by NathanLovato** [comment]
The shortcuts are still working for me, Ctrl+1/2/3.

i invite you to ensure you have the version of power sequencer included in Blender active. Otherwise, to add shortcuts manually, create a new shortcut in `SequencerCommon -> Sequencer -> Sequencer (global)` with the command `power_sequencer.playback_speed_set`

![image](https://user-images.githubusercontent.com/12694995/231701866-bb320821-4803-4a1c-8f58-0f32924c8b31.png)

**Comment by EroAxe** [comment]
Can confirm my Power Sequencer is enabled and they still don't work
![image](https://user-images.githubusercontent.com/29590243/231727665-61be76ab-6479-4af3-a219-390d0317d245.png)
They are also set just to clarify.  I also double checked and made sure that the other addons I had for the VSE were disabled to make sure.
![image](https://user-images.githubusercontent.com/29590243/231728070-08a91e73-0afc-48a1-91ee-f8b9ebf80925.png)

Besides that though, apologies for not being specific enough, but those are not the hotkeys I was referring to, specifically I am talking about these hotkeys for specifically increasing or decreasing the playback speed incrementally and are still listed in the documentation:
![image](https://user-images.githubusercontent.com/29590243/231727425-701d19a2-52f1-4e51-8404-46000153a130.png)
But seem to have disappeared in the last few versions version, even in the keymap.  Along with the other options I mentioned of 1.33x and 1.66x mentioned in the documentation there.  As I said, I relied relatively heavily on them and they've just been, gone for a bit.

**Comment by NathanLovato** [comment]
Okay then it's just that it's not possible to make this work in recent blender versions, this feature relies on a hack whose support was removed. Now this code just makes playback much slower. So indeed the docs should be updated, it's just not possible to support right now, unfortunately.

**Comment by EroAxe** [comment]
Welp, that's unfortunate.  Is there any way to at least get the increase/decrease hotkeys back for the current 1x/2x/3x options?

**Comment by NathanLovato** [comment]
No, as basically you'll only toggle between 1x and 2x speed typically, the shortcuts for increasing vs decreasing speed are gone.
